be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/RollCallValidator.scala

```diff
@@ -34,38 +38,33 @@ case object RollCallValidator extends MessageDataContentValidator {
     }
   }
 
-  // TODO check that this is correct (correct validations)
-  def validateOpenRollCall(rpcMessage: JsonRpcRequest): GraphMessage = {
-    def validationError(reason: String): PipelineError = super.validationError(reason, "OpenRollCall", rpcMessage.id)
+  def validateOpenRollCall(rpcMessage: JsonRpcRequest, validatorName: String = "OpenRollCall"): GraphMessage = {
+    def validationError(reason: String): PipelineError = super.validationError(reason, validatorName, rpcMessage.id)
 
     rpcMessage.getParamsMessage match {
       case Some(message: Message) =>
         val data: OpenRollCall = message.decodedData.get.asInstanceOf[OpenRollCall]
 
-        if (!validateTimestampStaleness(data.start)) {
-          Right(validationError(s"stale 'start' timestamp (${data.start})"))
+        val laoId: Hash = rpcMessage.extractLaoId
+        // Note: this id doesnt follow the standard sh256(string, hash, timestamp, string) other events do;
+        // this is why generateValidationId cannot be used
```

Comment:
If you need to write such a comment, it's one more reason to drop the `generateValidationId` and just use Hash everywhere ;-)